client/src/app/projects/[slug]/layout.tsx

```diff
@@ -1,14 +1,11 @@
 'use client';
 
-import Panel from '@/containers/panel';
 import ProjectDetailPanel from '@/containers/projects/detail/panel';
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <div>
-      <Panel>
```

Comment:
This dont works as Panel have children and you are injecting children directly in page.